packages/core/src/components/event-list-item/event-list-item.scss

```diff
@@ -82,6 +82,7 @@ $default-event-list-item-height: 2.5rem;
       max-height: var(--event-list-item-height, 100%);
       white-space: var(--event-list-item-content-white-space, inherit);
       overflow: hidden;
+      padding-right: $small-space;
```

Comment:
```suggestion
      padding-inline-end: $small-space;
```

Reply:
Looks to me like this change does not have any effect on the rendered component.
Also font truncation is not working in event list item currently. Maybe we can fix this with this PR?

Reply:
> padding-inline-end: $small-space;
With this change the icon will also be have the padding, i dont know if this is wanted.